doc/source/dev/contributor/ubuntu_guide.rst

```diff
@@ -1,51 +1,165 @@
 :orphan:
 
-.. _quickstart-ubuntu:
+.. _ubuntu-guide:
 
-=======================================================
-Development environment quickstart guide (Ubuntu)
-=======================================================
+======================================
+Development environment guide (Ubuntu)
```

Comment:
This guide is too specific and long, I recommend not spending more effort on it. We can refactor it away in a second pass over this content (too much for this PR probably).

Reply:
Yes - I'll accept the straightforward fixes but will leave this one for later.